nemesis-generator/scripts/ci/test.sh

```diff
@@ -6,3 +6,8 @@ TEST_RUNNER=$([ "$1" = "code-coverage" ] && echo "coverage run --append" || echo
 ${TEST_RUNNER} -m configuration_generator --count 20 --seed 450000000000000 --network-name testnet --output nemesis.yaml --accounts-output user.yaml
 ${TEST_RUNNER} -m generator --input ./nemesis.yaml --output nemesis.bin
 ${TEST_RUNNER} -m node_configuration_generator --accounts-file user.yaml --nodes-file resources/nodes.yaml --nemesis-file ./nemesis.yaml --seed ./nemesis.bin --network-friendly-name ship --output-path ./output
+
+# custom network
+${TEST_RUNNER} -m configuration_generator --count 20 --seed 450000000000000 --network-name testnet --network-identifier 136 --output nemesis.yaml --accounts-output user.yaml
```

Comment:
compared to above, this is the only different command (with --network-identifier)

Reply:
Yes, just added the custom network id scenario to verify it works.  doesnt do much for code coverage.